assets/README.md

```diff
@@ -29,6 +29,10 @@ This will build a complete bundle, including JavaScript and CSS.
 Using the flag `--type production` will result in minified JavaScript and CSS
 bundles.
 
+### `epub-build`
```

Comment:
Would it possible to have `build` do both `epub` and `html` builds? This way we should still keep `build` but we can then use the `epub-build` and `html-build` only if I want explicitly too.